apps/api/src/server/applications/application/documents/document.service.js

```diff
@@ -597,7 +599,7 @@ export const markDocumentVersionAsPublished = async ({
 export const markDocumentVersionAsUnpublished = async ({ guid, version }) => {
 	const publishedDocument = await documentVersionRepository.update(guid, {
 		version,
-		publishedStatus: 'unpublished',
+		publishedStatus: 'not_checked',
```

Comment:
I think we're required to set this back to the previous version, right @francescobrenci-AR ?

I'm just thinking - we currently set the previous version to 'ready_to_publish' before setting it to 'publishing' and then 'published' so our current plan might not work.

Reply:
good spot - had a chat with @HadrianHughesKC and @DavidJWhiteDev. We have a solution for this.

Versions that are unpublished will revert back to `not_checked` which will match the requirements on https://pins-ds.atlassian.net/browse/BOAS-1218